src/Toolkit/Toolkit.iOS/UI/Controls/SymbolDisplay/SymbolDisplay.iOS.cs

```diff
@@ -27,8 +27,8 @@ namespace Esri.ArcGISRuntime.Toolkit.UI.Controls
     [Category("ArcGIS Runtime Controls")]
     public partial class SymbolDisplay : IComponent
     {
-        private UIStackView _rootStackView;
-        private UIImageView _imageView;
+        private UIStackView? _rootStackView;
+        private UIImageView? _imageView;
```

Comment:
Would `[MemberNotNull]` work for these fields? We can get rid of several null-checks if that would work.